src/wobjectcpp.h

```diff
@@ -0,0 +1,124 @@
+#pragma once
+#include "wobjectdefs.h"
+
+namespace w_cpp {
+
+using w_internal::StringView;
+using w_internal::viewLiteral;
+
+template<typename T>
+constexpr auto makeProperty(StringView name, StringView type) {

```

Comment:
all the makeXXX needs documentation.  And also StringView

Reply:
done